packages/react-components/src/templates/DiscoverPageBody.tsx

```diff
@@ -54,10 +45,31 @@ const DashboardPageBody: React.FC<DashboardPageBodyProps> = ({
       </section>
     ) : null}
     {members.length ? (
-      <TeamMembersSection title={'Meet the ASAP team'} members={members} />
+      <TeamMembersSection
+        title={'Meet the ASAP team'}
+        members={members.map(
+          ({
+            displayName,
+            jobTitle,
+            institution,
+            firstName,
+            lastName,
+            avatarUrl,
+            id,
+          }) => ({
+            firstLine: displayName,
+            secondLine: jobTitle || '',
```

Comment:
`secondLine` supports an undefined value. Why do you default to an empty string?